src/main/java/Main.java

```diff
@@ -0,0 +1,52 @@
+import db.Storage;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import model.FruitTransaction;
+import service.CsvFileReaderService;
+import service.CsvFileWriterService;
+import service.FruitService;
+import service.FruitTransactionParser;
+import service.impl.CsvFileReaderServiceImpl;
+import service.impl.CsvFileWriterServiceImpl;
+import service.impl.FruitServiceImpl;
+import service.impl.FruitTransactionParserImpl;
+import strategy.OperationHandler;
+import strategy.handlers.BalanceHandler;
+import strategy.handlers.PurchaseHandler;
+import strategy.handlers.ReturnHandler;
+import strategy.handlers.SupplyHandler;

```

Comment:
The classes should be organized into packages according to their functionality to follow good code structuring practices. For example, `Storage` should be in the `db` package, `FruitTransaction` should be in the `model` package, services should be in the `service` package, their implementations in `service.impl`, and operation handlers in the `strategy` package.